### PR TITLE
Modify the method for determining whether a parameter is null

### PR DESCRIPTION
**Pull request by 1103283969**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by 1103283969** [comment]
Modify the method for determining whether a parameter is null

File: shenyu-plugin/shenyu-plugin-logging-rocketmq/src/main/java/org/apache/shenyu/plugin/logging/AbstractLogCollector.java [hunk 1]
```diff
@@ -61,9 +62,13 @@ public AbstractLogCollector(final LogConsumeClient logCollectClient) {
 
     @Override
     public void collect(final ShenyuRequestLog log) {
-        if (log == null || logCollectClient == null) {
+        /*if (log == null || logCollectClient == null) {
+            return;
+        }*/
+        if(Objects.nonNull(log)||Objects.nonNull(logCollectClient)){
```
[the remark this answers is not in the document]
Comment (yu199195) [reply shenyu-plugin/shenyu-plugin-logging-rocketmq/src/main/java/org/apache/shenyu/plugin/logging/AbstractLogCollector.java:68]: 1. you can exe mvn clean install in you local evn:
2. Objects.nonNull(log)  is mean  log == null?


like this: 
if (Objects.isNull(log) || Objects.isNull(logCollectClient)) {

}

**Comment by 1103283969** [comment]
Modify the method for determining whether a parameter is null

File: pom.xml [hunk 2]
```diff
@@ -143,6 +143,8 @@
         <spring-cloud-alibaba.version>2.2.3.RELEASE</spring-cloud-alibaba.version>
         <netflix-ribbon.version>2.2.9.RELEASE</netflix-ribbon.version>
         <dockerfile-maven-plugin.version>1.4.6</dockerfile-maven-plugin.version>
+        <maven.test.skip>true</maven.test.skip>
```
Comment (yu199195) [inline pom.xml:146]: remove this

File: shenyu-plugin/shenyu-plugin-logging-rocketmq/src/main/java/org/apache/shenyu/plugin/logging/AbstractLogCollector.java [hunk 3]
```diff
@@ -61,7 +62,7 @@ public AbstractLogCollector(final LogConsumeClient logCollectClient) {
 
     @Override
     public void collect(final ShenyuRequestLog log) {
-        if (log == null || logCollectClient == null) {
+        if (!Objects.nonNull(log) || !Objects.nonNull(logCollectClient)) {
```
Comment (yu199195) [inline shenyu-plugin/shenyu-plugin-logging-rocketmq/src/main/java/org/apache/shenyu/plugin/logging/AbstractLogCollector.java:65]: maybe used Objects.isNull

**Comment by 1103283969** [comment]
Modify the method for determining whether a parameter is null

**Review by yu199195 — APPROVED** [review]

**Pull request merged**